custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -129,6 +128,15 @@ def get_person_case_structure(self):
             },
         }
 
+        if self._location:
+            assert self._location.location_type.name == 'phi'
+            kwargs['attrs']['owner_id'] = self._location.location_id
+        else:
+            kwargs['attrs']['owner_id'] = '_archive_'
```

Comment:
I think it might be better if you use the constant defined in `casexml.apps.case.const`